packages/big-design/src/components/ButtonGroup/styled.tsx

```diff
@@ -0,0 +1,57 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { StyleableButton } from '../Button/private';
+import { FlexItem } from '../Flex';
+
+interface StyledButtonProps {
+  borderRadius?: boolean;
+}
+
+interface StyledFlexItemProps {
+  isVisible: boolean;
+}
+
+export const StyledButton = styled(StyleableButton)<StyledButtonProps>`
+  ${({ borderRadius, theme }) =>
+    borderRadius
+      ? css`
+          border-radius: ${theme.borderRadius.normal};
+        `
+      : css`
+          border-radius: ${theme.borderRadius.none};
+        `}
+
+  &:focus {
+    z-index: 1;
+  }
+`;
+
+export const StyledFlexItem = styled(FlexItem)<StyledFlexItemProps>`
+  margin-right: -1px;
+
+  &:last-of-type {
+    margin-right: 0;
+  }
+
+  &:first-of-type ${StyledButton} {
+    border-bottom-left-radius: ${({ theme }) => theme.borderRadius.normal};
+    border-top-left-radius: ${({ theme }) => theme.borderRadius.normal};
+  }
+
+  &:nth-last-of-type(-n + 2) ${StyledButton} {
```

Comment:
I use `&:nth-last-of-type(-n + 2)` instead of `:last-of-type` according to case when dropdown visible when some of actions has `actionType="destructive"` and all action are fit... so, it sets border radius to `dropdown toggle` and the last element of action.